docs/application/native/guides/text-input/tts.md

```diff
@@ -686,6 +719,38 @@ To start, pause, and stop the playback, follow the steps below:
   }
   ```
 
+<a name="playing_mode"></a>
+## Playing Mode
+
+Playing mode determines which process, either TTS service or TTS client, plays the synthesized PCM data. There're 2 different playing modes. They are described below:
```

Comment:
```suggestion
Playing mode determines which process, either TTS service or TTS client, plays the synthesized PCM data. Described below are the 2 different types of playing modes:
```

Reply:
Thank you for fixing it.
I'll change it according to your comment.